src/components/common/Weather/Weather.tsx

```diff
@@ -0,0 +1,37 @@
+import { FC } from 'react';
+import { TWeather} from "../../types/types";
+import css from './styles.module.css';
+
+interface WeatherProps {
+  weatherInfo: TWeather;
+  unit: 'standard' | 'metric' | 'imperial';
+}
+
+export const Weather: FC<WeatherProps> = ({weatherInfo, unit}) => {
+  return (
+    <table className={ css.table }>
+      <thead>
+      <tr className={ css.cityName }><th>{weatherInfo.name}</th></tr>
+      </thead>
+      <tbody>
+        <tr className={ css.weatherProp }>
+          <th className={ css.weatherImg }><img src={`https://openweathermap.org/img/wn/${ weatherInfo.weather[0].icon }@2x.png`} alt={'weather icon'}/></th>
+          <th className={ css.weatherTemp }>{weatherInfo.main.temp}{unit === 'metric' ? ' °C' : unit === 'imperial' ? ' °F' : ' K'}</th>
+        </tr>
+        <tr className={ css.weatherProp }>
+          <th>Feels like</th>
+          <th>{weatherInfo.main['feels_like']}{unit === 'metric' ? ' °C. ' : unit === 'imperial' ? ' °F. ' : ' K. '}</th>
+          <th>{weatherInfo.weather[0].description[0].toUpperCase() + weatherInfo.weather[0].description.slice(1)}</th>
+        </tr>
+        <tr className={ css.weatherProp }>
+          <th>Pressure:</th>
+          <th>{weatherInfo.main.pressure}</th>
+        </tr>
+        <tr className={ css.weatherProp }>
```

Comment:
prop это сокращение от пропсов? тогда оно не сосем уместно тут. У тебя на таблице уже висит стиль css.table, так то тут логично будет просто row использовать
```suggestion
        <tr className={ css.row}>
```